test/proc/proc_test.go

```diff
@@ -78,7 +90,12 @@ func TestIdentifyTCPCommunication(t *testing.T) {
 	if err := cli.ContainerStart(ctx, apiResp.ID, types.ContainerStartOptions{}); err != nil{
 		t.Fatal(err)
 	}
-	defer cli.ContainerStop(ctx, apiResp.ID, nil)
+	defer func(){
+		err := cli.ContainerStop(ctx, apiResp.ID, nil)
+		if err != nil {
+			t.Fatal(err)
+		}
+		}()
```

Comment:
```suggestion
　　err := cli.ContainerStop(ctx, apiResp.ID, nil)
　　if err != nil {
　　  t.Fatal(err)
　　  }
```
となにが違うんでしたっけ？軽く調べたんですがわかりませんでした...

Reply:
テスト終了時にコンテナの削除処理をしてほしいので，deferでContainerStopを実行するようにしていました．しかし，lint曰くerrorを返す関数の処理はエラーへの対応を行ってほしいということだったので，defer内にコンテナの削除処理とエラー表示を追加したという感じです…（表示するならFatalじゃなくて，Errorだよね…）

Reply:
あーごめんなさい
inspect, err := cli.ContainerInspect(ctx, apiResp.ID)を見落としてて、もうcli使わないならdefer使わず閉じても良いのでは？って思っていました

FatalかErrorかは難しいところですね テストの本質じゃないのでErrorでもいいような気もします